app/views/collection_items/_item_fields.html.erb

```diff
@@ -1,64 +1,63 @@
 <li class="collection item picture blurb group">
-  <div class="header module">
-  <% item_name = collection_item.item_type == "Bookmark" ? 
-            ts("Bookmark for %{title}", :title => collection_item.item.bookmarkable.title) : 
-            (collection_item.item.title + (!collection_item.item.posted? ? ts("(draft)") : "")) %>
-
   <%= fields_for("collection_items[]", collection_item) do |form| %>
-    <!--title-->
-    <h4 class="heading" id="collection_item_<%= collection_item.id %>">
-      <%= error_messages_for collection_item %>    
-      <%= form.hidden_field :collection_id %>
-      <%= link_to(item_name, collection_item.item) %>   
-    </h4>
+    <div class="header module">
+      <!--title-->
+      <h4 class="heading" id="collection_item_<%= collection_item.id %>">
+        <%= error_messages_for collection_item %>
+        <%= form.hidden_field :collection_id %>
+        <%= link_to collection_item_display_title(collection_item), collection_item.item %>
+      </h4>
 
-    <% unless @user %>
-      <% # mod version %>
       <h5 class="heading">
-        <% collection_item.item_creator_pseuds.each do |pseud| %>
-          <%= pseud.byline %>
+        <% if @user %>
+          <% # user version %>
+          <%= ts("in") %> <span class="collection"><%= link_to collection_item.collection.title, collection_path(collection_item.collection) %></span>
+          <% if collection_item.collection.user_is_posting_participant?(@user) %>
+            <%= ts("(Member)") %>
+          <% end %>
+        <% else %>
+          <% # mod version %>
+          <% collection_item.item_creator_pseuds.each do |pseud| %>
+            <%= pseud.byline %>
             <% if collection_item.collection.user_is_posting_participant?(pseud.user) %>
               <%= ts("(Member)")%>
             <% end %>
           <% end %>
-        <% # TODO: Make recipients and bylines links, separate recipients with space (currently "snow,astolat") %>
-        <% unless collection_item.recipients.blank? %>
-          <span class="recipients"><%= ts("for") %> <span class="user"><%= collection_item.recipients %></span></span>
+          <% # TODO: Make recipients and bylines links, separate recipients with space (currently "snow,astolat") %>
```

Comment:
Would calling `to_sentence` on `collection_item.recipients` take care of this TODO ?